.github/workflows/release.yaml

```diff
@@ -1,375 +0,0 @@
-name: Release
-on:
-  workflow_dispatch:
-    inputs:
-      version:
-        type: string
-        description: The version you intend to release without leading 'v' (eg x.y.z and not vx.y.z)
-  pull_request:
-    types: [closed]
-  push:
-    branches:
-      - "release/**"
-      - "next/**"
-    tags:
-      - v*
-  release:
-    types: [created]
-
-env:
-  VERSION: ${{ github.event.inputs.version }}
-  APP_ID: 251311
-
-jobs:
-  prepare:
-    runs-on: ubuntu-latest
-    if: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.version != '' }}
-    steps:
-      - name: Generate token
-        id: generate_token
-        uses: tibdex/github-app-token@3beb63f4bd073e61482598c45c71c1019b59b73a
```

Comment:
This isn't always needed, we can simply use `secrets.GITHUB_TOKEN` in most cases. One exception is `create-release-pr.yaml`, where _the action_ creates a PR and another checks needs to be triggered by this PR. In this case, we can use the official github action for this purpose.